### PR TITLE
18). Add topic files 43 to 46

### PR DESCRIPTION
**Pull request by jameshkramer**
@mairaw @BillWagner
This PR is for user story #1576037.

It adds four topic files, makes corresponding deletions from the C# and VB folders, and adds redirection entries. There is also a redirections fix.

cc: @tfosmark @marteeleigh

Contributes to #4728

### CODE REVIEW

**Review by BillWagner — APPROVED** [review]
This LGTM @jameshkramer 

@mairaw Do you have anything to add?

File: docs/standard/linq/find-element-specific-attribute.md [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+ms.date: 07/20/2015
+dev_langs:
+  - "csharp"
+  - "vb"
+ms.assetid: b92591aa-3cfb-490e-99f6-da8de335e362
+---
+
+# How to find an element with a specific attribute (LINQ to XML)
+
+This topic provides examples of how to find an element whose attribute has a specific value.
```
Comment (mairaw) [inline docs/standard/linq/find-element-specific-attribute.md:12]:
```suggestion
This article provides examples of how to find an element whose attribute has a specific value.
```

File: docs/standard/linq/query-xml-trees-overview.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+# Query XML trees overview (LINQ to XML)
+
+After you have instantiated an XML tree, writing queries is the most effective way to extract data from the tree. Also, querying combined with functional construction enables you to generate a new XML document that has a different shape from the original document.
+
+For background information about LINQ queries, see:
+
+- [Introduction to LINQ Queries (C#)](../../csharp/programming-guide/concepts/linq/introduction-to-linq-queries.md)
+- [Writing Your First LINQ Query (Visual Basic)](../../visual-basic/programming-guide/concepts/linq/writing-your-first-linq-query.md)
+
+## In This Section
```
Comment (mairaw) [inline docs/standard/linq/query-xml-trees-overview.md:16]:
```suggestion
## In this section
```

File: docs/standard/linq/query-xml-trees-overview.md [hunk 3]
```diff
@@ -0,0 +1,35 @@
+---
+title: Query XML trees overview - LINQ to XML
+ms.date: 07/20/2015
+ms.assetid: 2e35c1ab-08c8-4378-9ca8-8ff344756eda
+---
+
+# Query XML trees overview (LINQ to XML)
+
+After you have instantiated an XML tree, writing queries is the most effective way to extract data from the tree. Also, querying combined with functional construction enables you to generate a new XML document that has a different shape from the original document.
```
Comment (mairaw) [inline docs/standard/linq/query-xml-trees-overview.md:9]:
```suggestion
After you've instantiated an XML tree, writing queries is the most effective way to extract data from the tree. Also, querying combined with functional construction enables you to generate a new XML document that has a different shape from the original document.
```

**Review by mairaw — APPROVED** [review]
Looks good. Left a few nit comments.

**Pull request merged**